protos/local/temporal/sdk/core/activity_result/activity_result.proto

```diff
@@ -1,6 +1,7 @@
 syntax = "proto3";
 
 package coresdk.activity_result;
+option ruby_package = "Coresdk::ActivityResult";
```

Comment:
Wouldn't you want this something like `Temporalio::Bridge::Proto::ActivityResult`?

Reply:
I agree that would be nicer.

Right now TS and Ruby both use the "official" package name, except for camel case in Ruby. Is that what they are named in .Net and Python?

Reply:
Mostly. It is `temporalio.bridge.proto.activity_result` in Python and `Temporalio.Bridge.Api.ActivityResult` in .NET. I find putting everything under "bridge" and making all that internal/private and/or excluded from docs is helpful.

Reply:
Updated.